diff --git a/model/VQGAN/vqgan.py b/model/VQGAN/vqgan.py
index d5f0603..1c07596 100644
--- a/model/VQGAN/vqgan.py
+++ b/model/VQGAN/vqgan.py
@@ -61,7 +61,7 @@ class VQModel(pl.LightningModule):
             self.monitor = monitor
 
     def init_from_ckpt(self, path, ignore_keys=list()):
-        sd = torch.load(path, map_location="cpu")["state_dict"]
+        sd = torch.load(path, map_location="cpu", weights_only=False)["state_dict"]
         keys = list(sd.keys())
         for k in keys:
             for ik in ignore_keys:
diff --git a/runners/BaseRunner.py b/runners/BaseRunner.py
index 8236a04..7cd3ff5 100755
--- a/runners/BaseRunner.py
+++ b/runners/BaseRunner.py
@@ -102,7 +102,7 @@ class BaseRunner(ABC):
         model_states = None
         if self.config.model.__contains__('model_load_path') and self.config.model.model_load_path is not None:
             print(f"load model {self.config.model.model_name} from {self.config.model.model_load_path}")
-            model_states = torch.load(self.config.model.model_load_path, map_location='cpu')
+            model_states = torch.load(self.config.model.model_load_path, map_location='cpu', weights_only=False)
 
             self.global_epoch = model_states['epoch']
             self.global_step = model_states['step']
@@ -128,7 +128,7 @@ class BaseRunner(ABC):
             # load optimizer and scheduler
             if self.config.args.train:
                 if self.config.model.__contains__('optim_sche_load_path') and self.config.model.optim_sche_load_path is not None:
-                    optimizer_scheduler_states = torch.load(self.config.model.optim_sche_load_path, map_location='cpu')
+                    optimizer_scheduler_states = torch.load(self.config.model.optim_sche_load_path, map_location='cpu', weights_only=False)
                     for i in range(len(self.optimizer)):
                         self.optimizer[i].load_state_dict(optimizer_scheduler_states['optimizer'][i])
 
@@ -347,34 +347,34 @@ class BaseRunner(ABC):
             test_sampler = torch.utils.data.distributed.DistributedSampler(test_dataset)
             train_loader = DataLoader(train_dataset,
                                       batch_size=self.config.data.train.batch_size,
-                                      num_workers=0,
+                                      num_workers=8,
                                       drop_last=True,
                                       sampler=train_sampler)
             val_loader = DataLoader(val_dataset,
                                     batch_size=self.config.data.val.batch_size,
-                                    num_workers=0,
+                                    num_workers=8,
                                     drop_last=True,
                                     sampler=val_sampler)
             test_loader = DataLoader(test_dataset,
                                      batch_size=self.config.data.test.batch_size,
-                                     num_workers=0,
+                                     num_workers=8,
                                      drop_last=True,
                                      sampler=test_sampler)
         else:
             train_loader = DataLoader(train_dataset,
                                       batch_size=self.config.data.train.batch_size,
                                       shuffle=self.config.data.train.shuffle,
-                                      num_workers=0,
+                                      num_workers=8,
                                       drop_last=True)
             val_loader = DataLoader(val_dataset,
                                     batch_size=self.config.data.val.batch_size,
                                     shuffle=self.config.data.val.shuffle,
-                                    num_workers=0,
+                                    num_workers=8,
                                     drop_last=True)
             test_loader = DataLoader(test_dataset,
                                      batch_size=self.config.data.test.batch_size,
                                      shuffle=False,
-                                     num_workers=0,
+                                     num_workers=8,
                                      drop_last=True)
 
         epoch_length = len(train_loader)
diff --git a/runners/DiffusionBasedModelRunners/BBDMRunner.py b/runners/DiffusionBasedModelRunners/BBDMRunner.py
index b9c18e3..87608d1 100755
--- a/runners/DiffusionBasedModelRunners/BBDMRunner.py
+++ b/runners/DiffusionBasedModelRunners/BBDMRunner.py
@@ -32,7 +32,7 @@ class BBDMRunner(DiffusionBaseRunner):
         states = None
         if self.config.model.only_load_latent_mean_std:
             if self.config.model.__contains__('model_load_path') and self.config.model.model_load_path is not None:
-                states = torch.load(self.config.model.model_load_path, map_location='cpu')
+                states = torch.load(self.config.model.model_load_path, map_location='cpu', weights_only=False)
         else:
             states = super().load_model_from_checkpoint()
 
@@ -66,7 +66,7 @@ class BBDMRunner(DiffusionBaseRunner):
         else:
             scheduler = torch.optim.lr_scheduler.ReduceLROnPlateau(optimizer=optimizer,
                                                                 mode='min',
-                                                                verbose=True,
+
                                                                 threshold_mode='rel',
                                                                 **vars(config.model.BB.lr_scheduler)
 )
